apps/src/code-studio/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -50,13 +50,20 @@ const WorkshopAttendance = React.createClass({
       loading: true,
       workshopState: undefined,
       sessionAttendances: undefined,
-      adminActions: undefined,
       adminOverride: false,
       saving: false,
       isModified: false
     };
   },
 
+  isAdmin() {
+    return window.dashboard.workshop.permission === "admin";
```

Comment:
Yes, it is admin status. See https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/pd/workshop_dashboard_controller.rb#L7-L15

I need to differentiate between admin (manage all workshops), workshop organizer (manage my own workshops), and facilitator (view and update attendance on my workshops). Ideally, instead of `User.admin?` we could have a `UserPermission::workshop_admin` or something similar. Would you prefer that? Otherwise do you have a better suggestion for the variable name?